modules/UI/toolbars/Toolbar.js

```diff
@@ -6,7 +6,68 @@ import SideContainerToggler from "../side_pannels/SideContainerToggler";
 let emitter = null;
 let Toolbar;
 
+/**
+ * Opens the invite link dialog.
+ */
+function openLinkDialog () {
```

Comment:
This code reverts a change that has been done in the latest invite dialog code. You need to re-merge your code with the latest master and have a look at the changes done by @BeatC on invite dialogs.